apps/staff/components/request-management.tsx

```
@@ -287,6 +278,28 @@ export function RequestManagement({ request, onStatusUpdate }: RequestManagement
                         )}
                       </div>
+                      {request.formData && Object.keys(request.formData).length > 0 && (
+                        <div className="bg-gray-50 p-4 rounded-lg">
+                          <h4 className="font-medium mb-2">
+                            {REQUEST_TYPE_LABELS[request.type] || request.type.replace(/_/g, ' ')} —
+                            Submission Details
+                          </h4>
+                          <div className="space-y-2">
+                            {getFormDataDisplayItems(request.type, request.formData).map(
+                              (item, index) => (
+                                <div key={index}>
```

Biome (2.4.10)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)